plugins/discourse-presence/assets/javascripts/discourse/components/composer-presence-display.js.es6

```diff
@@ -15,115 +11,72 @@ export default Component.extend({
   topic: null,
   reply: null,
   title: null,
+  isWhispering: null,
 
-  // Internal variables
-  previousState: null,
-  currentState: null,
-  presenceUsers: null,
-  channel: null,
-
+  presenceManager: alias("topic.presenceManager"),
```

Comment:
Generally speaking, I'm not a big fan of alias, in your case I think https://api.emberjs.com/ember/3.17/functions/@ember%2Fobject%2Fcomputed/readOnly is a better choice, given you don't want to do set on these aliased properties